.github/workflows/fixFill.yaml

```diff
@@ -0,0 +1,32 @@
+name: Fix SVG fill
```

Comment:
I think that this action is a bad idea. On every push to icons there will be the commit with fixing SVG fill. Leaving aside that the commit message does not meet our standards I suggest running the normalizing script right before publishing, eg. please, add `prebuild` command which will do this heavy lifting instead of GitHub Actions.

Reply:
Ok, I removed all commits except the supernova one. We can merge this and I will prepare a new branch with these changes.

Reply:
👌